core/src/main/java/org/apache/carbondata/core/metadata/converter/ThriftWrapperSchemaConverterImpl.java

```diff
@@ -381,6 +422,40 @@ private DataType fromExternalToWrapperDataType(org.apache.carbondata.format.Data
     return wrapperColumnSchema;
   }
 
+  private PartitionType fromExternalToWrapperPartitionType(
+      org.apache.carbondata.format.PartitionType externalPartitionType) {
+    if (null == externalPartitionType) {
+      return null;
+    }
+    switch (externalPartitionType) {
+      case HASH:
+        return PartitionType.HASH;
+      case LIST:
+        return PartitionType.LIST;
+      case RANGE:
+        return PartitionType.RANGE;
+      case RANGE_INTERVAL:
+        return PartitionType.RANGE_INTERVAL;
+      default:
+        return PartitionType.HASH;
+    }
+  }
+
+  private PartitionInfo fromExternalToWrapperPartitionInfo(
+      org.apache.carbondata.format.PartitionInfo externalPartitionInfo) {
+    List<ColumnSchema> wrapperColumnSchema = new ArrayList<ColumnSchema>();
+    for (org.apache.carbondata.format.ColumnSchema columnSchema : externalPartitionInfo
```

Comment:
move `externalPartitionInfo` to next line, please also check other newly added code in this class